src/back/handlers.py

```diff
@@ -0,0 +1,20 @@
+"""Module containing handlers."""
+from typing import Self
+
+from tornado.iostream import IOStream
+
+from src.back.interfaces.handlers import CreateRoomHandlerInterface
+from src.back.interfaces.message import MessageInterface
+from src.back.interfaces.room_registry import RoomRegistryInterface
+from src.back.room import Room
+
+
+class CreateRoomHandler(CreateRoomHandlerInterface):
+    """Interface for create room."""
+
+    room_registry: RoomRegistryInterface
+
+    def handle(self: Self, message: MessageInterface, stream: IOStream) -> None:
+        """Create room."""
+        room = Room(name=message.from_bytes())
```

Comment:
how do i instantiate room from message?

Reply:
Just use a concrete implementation of the message:
```python
class CreateRoomMessage(MessageInterface):
    name: str


class CreateRoomHandler(CreateRoomHandlerInterface):
    """Interface for create room."""

    room_registry: RoomRegistryInterface

    def handle(self: Self, message: CreateRoomMessage, stream: IOStream) -> None:
        """Create room."""
        room = Room(name=message.name)

```